yak_ros/CMakeLists.txt

```diff
@@ -150,13 +150,20 @@ endif()
 if ($ENV{ROS_VERSION} VERSION_EQUAL "1")
     set(ROS_LIB_DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
     set(ROS_SHARE_DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
One additional thing to add: You can delete these two lines, since you're directly using `CATKIN_PACKAGE_LIB_DESTINATION`, etc. in the CMake `install` function. The original purpose was to be able to use the same `ROS_LIB_DESTINATION`, etc., variables for both ROS1 and ROS2, but that isn't really applicable as currently written.

Reply:
I did not remove `set(ROS_SHARE_DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})`, as it is still being used to set the install location for the launch files.

I did remove `set(ROS_LIB_DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})`.   However, since the install rule for the demo was still using it, I moved that inside the if(ROS1) area and updated it to use the 3-line install rule.  My mistake for not noticing that with the first commit.
```
    if (DEFINED BUILD_DEMO)
        install(TARGETS ${PROJECT_NAME}_image_simulator
```

Reply:
OK, good catch on the use of `ROS_SHARE_DESTINATION` further down.